Storage/SQL/RemoteTabsTable.swift

```diff
@@ -96,7 +98,8 @@ class RemoteTabsTable<T>: GenericTable<RemoteTab> {
     override func getUpdateAndArgs(inout item: RemoteTab) -> (String, [AnyObject?])? {
         var args = [AnyObject?]()
         args.append(item.title)
-        args.append(nil) // TODO: persist history.
+        let historyString = ",".join(item.history.filter { $0.absoluteString != nil }.map { $0.absoluteString! })
```

Comment:
Commas *sometimes* occur in real-world URLs, even though RFCs say they shouldn't. The safest thing to do here is to turn `item.history` into a JSON string, or null- or newline-separate it, rather than using commas.